### PR TITLE
Feat test networks remove

### PR DESCRIPTION
**Pull request by asimpk**

### CODE REVIEW

File: packages/lib-ethers/src/EthersKumoConnection.ts [hunk 1]
```diff
@@ -4,10 +4,6 @@ import { Signer } from "@ethersproject/abstract-signer";
 import { Decimal } from "@kumodao/lib-base";
 
 import devOrNull from "../deployments/dev.json";
```
Comment (udosson) [inline packages/lib-ethers/src/EthersKumoConnection.ts:6]: please add 
`import devOrNull from "../deployments/mumbai.json";`

File: packages/lib-ethers/hardhat.config.ts [hunk 2]
```diff
@@ -104,18 +97,20 @@ const config: HardhatUserConfig = {
...
+      throwOnTransactionFailures: false,
     },
 
     dev: {
       url: "http://localhost:8545",
       accounts: [deployerAccount, devChainRichAccount, ...generateRandomAccounts(numAccounts - 2)]
     },
+    mumbai: {
+      url: `https://matic-mumbai.chainstacklabs.com`,
+      accounts: ['ff126760c1b50be914c632a2cfbcbbfc569e21d07008c92c861e8977f0c01544'],
```
Comment (udosson) [inline packages/lib-ethers/hardhat.config.ts:109]: Remove private key and use `.env` file instead

File: packages/lib-ethers/hardhat.config.ts [hunk 3]
```diff
@@ -70,25 +70,18 @@ const oracleAddresses = {
...
-    chainlink: "0x9326BFA02ADD2366b30bacB125260Af641031331",
-    tellor: "0x20374E579832859f180536A69093A126Db1c8aE9" // Playground
-  }
 };
 
 const hasOracles = (network: string): network is keyof typeof oracleAddresses =>
   network in oracleAddresses;
 
 const wethAddresses = {
   mainnet: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
```
Comment (udosson) [inline packages/lib-ethers/hardhat.config.ts:83]: Please update `wethAddresses` for Polygon and Mumbai network

File: packages/lib-ethers/hardhat.config.ts [hunk 4]
```diff
@@ -70,25 +70,18 @@ const oracleAddresses = {
     chainlink: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419",
     tellor: "0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0"
   },
-  rinkeby: {
+  mumbai: {
```
Comment (udosson) [inline packages/lib-ethers/hardhat.config.ts:73]: update chainlink and tellor addresses, please.
This needs to be updated for Mumbai and Polygon PoS

**Review by udosson — CHANGES_REQUESTED** [review]
Thanks @asimpk - Just a few small changes ;)

File: packages/lib-ethers/hardhat.config.ts (hunk 4 above)
Comment (asimpk) [reply packages/lib-ethers/hardhat.config.ts:73]: Please verify are these addresses are correct?
` mainnet: {
    chainlink: "0xb33D8A4e62236eA91F3a8fD7ab15A95B9B7eEc7D",
    tellor: "0xe3322702bedaaed36cddab233360b939775ae5f1"
  },
  mumbai: {
    chainlink: "0x58bbdbfb6fca3129b91f0dbe372098123b38b5e9",
    tellor: "0x3477EB82263dabb59AC0CAcE47a61292f28A2eA7" // Core
  },`

Comment (udosson) [reply packages/lib-ethers/hardhat.config.ts:73]: @asimpk 
the Tellor address for mainnet is https://polygonscan.com/address/0xfd45ae72e81adaaf01cc61c8bce016b7060dd537#code

You can get the addresses for the chainlink data feeds for ETH/USD here:
Polygon Mainnet: https://docs.chain.link/docs/matic-addresses/#Polygon%20Mainnet
Mumbai Testnet: https://docs.chain.link/docs/matic-addresses/#Mumbai%20Testnet

**Review by udosson — APPROVED** [review]
Looks perfect - @asimpk. Thank you!
Let's merge this PR in our call today

**Pull request merged**